eo-runtime/src/main/java/org/eolang/ExprReduce.java

```diff
@@ -73,46 +71,44 @@
     /**
      * Ctor.
      *
-     * @param oper Operation name
-     * @param param Name of parameter with varargs
-     * @param type Type of varargs
+     * @param arguments First is operation name, second is name of parameter with varargs
      * @param reduction Reduction operation on consecutive varags
      * @param validation Validation function
      */
     public ExprReduce(
-        final String oper,
-        final String param,
-        final Class<T> type,
+        final String[] arguments,
```

Comment:
@potatmen with such change we still don't fix the problem but just hide it. `ExprReduce` still has too many arguments to operate with (even more now since array is unlimited).
I believe we need to extract some of the `ExprReduce` functionality into new object and use it as argument for `ExprReduce`. 
Something which would look like:
```
new ExprReduce<>(
    "oper name",
    new ExprReduce.Args(
        "x",
        Long.class,
        <validation lambda>
    ),
    <reduction lambda>
)
```
Method `ExprReduce.get()` should be amended accordingly to use new construct.